saleor/graphql/order/tests/queries/test_order_with_filter.py

```diff
@@ -1249,3 +1249,79 @@ def test_order_query_with_filter_not_allow_numbers_and_ids_together(
     # then
     assert content["errors"][0]["message"] == error_message
     assert not content["data"]["orders"]
+
+
+def test_order_query_with_filter_by_checkout_token(
+    orders_query_with_filter,
+    staff_api_client,
+    permission_manage_orders,
+    orders_from_checkout,
```

Comment:
I would also add some orders here that doesn't have a given token. Just to confirm that we don't return all(), but only the one that have a checkout.token provided.

Reply:
Actually it was done already, but thanks for that comment, now I realized that I shouldn't have included orders without token in fixture named this way.